cg/meta/sbatch.py

```diff
@@ -12,3 +12,8 @@
 ERROR_RSYNC_FUNCTION = """
 echo "Rsync failed"
 """
+
+QOS = {
```

Comment:
Could you use/add to the constants here: cg/constants/priority.py instead?

Reply:
you mean like this? (see changes)